### PR TITLE
fix: set a width on the cards for the favorite positions profile screen

### PR DESCRIPTION
**Pull request by rtirserio**
resolves TM-410

### CODE REVIEW

**Comment by mjoyce91** [comment]
We should update the favorite button and bid count to ignore responsiveness if the card is set to a fixed width:

screen shot 2019-02-15 at 9 28 22 am
screen shot 2019-02-15 at 9 28 28 am

**Comment by rtirserio** [comment]
@mjoyce91 the cards that im making fixed-width are only on the favorites page. Does it make sense to apply this to all the cards? Changing the responsive behavior of the button and bid counts would affect all of the cards.

**Comment by rtirserio** [comment]
@mjoyce91 - updated to not make the bid counts and favorites button responsive and applied fixed with to the cards on the homepage and similar positions. See TM-510

**Comment by rtirserio** [comment]
What are you referring g to specifically here?

**Comment by mjoyce91** [comment]
@rtirserio notice how the grid goes from 3 on top & 2 on the bottom, to 2 on top & 3 on the bottom?

**Comment by rtirserio** [comment]
@mjoyce91 fixed the wrapping issue

File: src/sass/_condensedCard.scss [hunk 1]
```diff
@@ -213,11 +213,16 @@ $condensed-card-data-padding: 15px 20px 9px;
 @media screen and (max-width: $screen-md-min) {
   .usa-width-one-third.condensed-card {
     display: block;
+
+    &:last-child {
+      margin-right: 4.82916%;
+    }
   }
 }
```
Comment (mjoyce91) [inline src/sass/_condensedCard.scss]: This was the right idea, but you just need one more rule:

```
@media screen and (min-width: $screen-lg-min) {
  .usa-width-one-third.condensed-card {
    margin-right: 2.35765%;
  }
}
```

**Comment by mjoyce91** [comment]
Added one comment, we should be good after adding it in

File: src/sass/_condensedCard.scss (hunk 1 above)
Comment (rtirserio) [reply src/sass/_condensedCard.scss]: Fixed. Thanks for the help, I ❤️ CSS

**Pull request merged**